### PR TITLE
Switch to ESM

### PR DESCRIPTION
**Pull request by katydecorah**
This PR will switch alt-text to an ES module and update eslint@8.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,16 +1,15 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```
Comment (katydecorah) [inline .eslintrc:2]: @jsnmrs I think we were seeing the errors with eslint 8, because babel-eslint is deprecated.

File: scripts/url.js [hunk 2]
```diff
@@ -31,7 +32,7 @@ const shortenUrls = async () => {
       return o;
     }, {});
 
-  fs.writeFileSync("urls.json", `${JSON.stringify(sorted, null, 2)}\n`);
+  writeFileSync("urls.js", `export default ${JSON.stringify(sorted, null, 2)}\n`);
```
Comment (katydecorah) [inline scripts/url.js:35]: Note: It's not as easy to import JSON files in ESM, so I switch urls.json to urls.js.

File: .eslintrc (hunk 1 above)
Comment (jsnmrs) [reply .eslintrc:2]: ah, that makes sense!

File: scripts/url.js (hunk 2 above)
Comment (jsnmrs) [reply scripts/url.js:35]: interesting — thanks!

**Review by jsnmrs — APPROVED** [review]

**Pull request merged**